lte/gateway/c/session_manager/SessionState.cpp

```diff
@@ -2826,4 +2841,60 @@ void SessionState::process_get_5g_rule_installs(
   }
 }
 
+void SessionState::process_get_mod_rule_installs(
+    const std::vector<QosPolicy>& qospolicy, RulesToProcess* pending_activation,
+    RulesToProcess* pending_deactivation) {
+  for (const QosPolicy& qos_policy : qospolicy) {
+    const std::string& rule_id = qos_policy.qos().id();
+    if (is_static_rule_installed(rule_id)) {
+      PolicyRule rule;
+      RuleToProcess to_process;
+      static_rules_.get_rule(rule_id, &rule);
+      to_process.version = get_current_rule_version(rule.id());
+      to_process.rule = rule;
+      to_process.teids = config_.common_context.teids();
+      if (qos_policy.policy_action() == QosPolicy::ADD) {
+        MLOG(MDEBUG) << " Static Rule pending_activation qos_policy.version :"
```

Comment:
why is this log here between ifs?

Reply:
For Testing purpose added this log, now I have removed these logs.